src/main/java/at/hannibal2/skyhanni/features/commands/WikiManager.kt

```diff
@@ -35,21 +29,25 @@ class WikiManager {
         if (!LorenzUtils.inSkyBlock) return
         if (!isEnabled()) return
 
+        val urlPrefix = if (config.useFandom) "https://hypixel-skyblock.fandom.com/wiki/" else "https://wiki.hypixel.net/"
+        val urlSearchPrefix = if (config.useFandom) "${urlPrefix}Special:Search?query=" else "${urlPrefix}index.php?search="
+
         val message = event.message.lowercase()
         if (!(message.startsWith("/wiki"))) return
         event.isCanceled = true
         if (message == "/wiki") {
-            LorenzUtils.chat("Opening the Fandom Wiki..")
-            OSUtils.openBrowser("${urlPrefix}Hypixel_SkyBlock_Wiki")
+            LorenzUtils.clickableLinkChat("Click to open the wiki!", urlPrefix, config.autoOpenWiki, "Open the wiki!")
         } else if (message.startsWith("/wiki ") || message == ("/wikithis")) { //conditional to see if we need Special:Search page
             if (message == ("/wikithis")) {
                 val itemInHand = InventoryUtils.getItemInHand() ?: return
-                wikiTheItem(itemInHand)
+                wikiTheItem(itemInHand, config.autoOpenWiki)
             } else {
                 val search = event.message.split("/wiki ").last()
```

Comment:
Why use split here? Even removePrefix would be better (but still flawed, since that one isn't case insensitive). Also *please* urlEncode the search.

```suggestion
                val search = event.message.drop("/wiki ".length)
```

Reply:
I didn't make this line.
I think @ErymanthusRayDeeUx did

Reply:
I didnt say you did it. Either way a semantically wrong thing in your pull request, so i will point it out.